app/MMB/src/__tests__/MMBContext.test.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+
+import * as React from 'react';
+import renderer from 'react-test-renderer';
+
+import MMBContext from '../MMBContext';
+
+const MMBContextConsumer = () => null;
+
+const ConsumerWithContext = () => (
+  <MMBContext.Consumer>
+    {({ accessToken, actions: { setAccessToken } }) => (
+      <MMBContextConsumer
+        accessToken={accessToken}
+        setAccessToken={setAccessToken}
+      />
+    )}
+  </MMBContext.Consumer>
+);
+
+describe('MMBContext', () => {
+  it('provides its state to consumers', () => {
+    const wrapper = renderer.create(<ConsumerWithContext />);
+    const consumerProps = wrapper.toTree().rendered.props;
+
+    expect(consumerProps.setAccessToken).toBeDefined();
+    expect(consumerProps.accessToken).toBe(null);
+  });
+
+  it('sets the accessToken', () => {
+    const wrapper = renderer.create(
+      <MMBContext.Provider>{null}</MMBContext.Provider>,
+    );
+    const instance = wrapper.getInstance();
+    instance.setAccessToken('myToken');
```

Comment:
@tbergq What about using context action directly? This way you'll actually test the real API because nobody is going to use it like this in real code...